backend/src/routes/volunteerRoutes.ts

```diff
@@ -1 +1,18 @@
-//volunteerRoutes.js
\ No newline at end of file
+import { Router, Request, Response } from 'express';
+
+import { 
+    getVolunteerById, 
+    getVolunteers, 
+    updateVolunteer 
+} from '../controllers/volunteerController.js';
+
+const router = Router();
+
+// get volunteer profile by id
+router.get('/volunteer/:volunteer_id', (req: Request, res: Response) => { getVolunteerById(req, res) });
+
+// get all volunteers
+router.get('/volunteers', (req: Request, res: Response) => { getVolunteers(req, res) });
+
+// update volunteer profile
+router.put('/volunteer/:volunteer_id', (req: Request, res: Response) => { updateVolunteer(req, res) });
```

Comment:
need `export default router` at the bottom of this file so we can define it in the index.ts file

Reply:
Then in src/index.ts we will need to define a parent path to all volunteer routes like so:

import volunteerRouter from './routes/volunteerRoutes.js';
...
app.use('/volunteer', volunteerRouter);

so all volunteer endpoints will have this parent path

Reply:
sure np